### ISSUE TITLE
License unsuitable for software

### ISSUE THREAD

**Issue by Wissperwind** [browserslist/caniuse-lite#106]
Hi,

Would you mind changing your license to one that is designed for software?
The chosen license makes it really hard to deside wether your library and ALL LIBRARIES USING YOUR LIBRARY can be used professionally. This would make the live of all who care about licenses mouch easier.

**Comment by ai** [comment]
I can’t.

This is a repack of [Can I Use package](npmjs.com/package/caniuse-db). They use CC. I must use it too.

What exact problem do you have? CC is a very well known license with many docs for edge cases.

**Comment by Wissperwind** [comment]
Some problems:

- Management did not know the license, thinking, better don't use it.
- Management opens the license site, there is written "not suitable for software", thinking, better don't use it.
- unclear question what "no futher regulaions" in the license shall mean, management thinks, better don't use it.
- software nearly finished, shall be shipped, but we have to investigate hours with the question wether or not we can use this license.
So in summary mouch work for us.

**Comment by ai** [comment]
> Management did not know the license, thinking, better don't use it.

Seems like it is a problem in legal knowledge in the management.

CC is very popular for icons and many other products. How they plan to develop software without knowledge of CC?

> Management opens the license site, there is written "not suitable for software", thinking, better don't use it.

Can I Use is not a software but set of JSON files. It is closer to WIkipedia.

> software nearly finished, shall be shipped, but we have to investigate hours with the question wether or not we can use this license.

You are using my (and many other people) work for free. Don’t use this card.

**Comment by Wissperwind** [comment]
OK, I have understood that you can not chose a different license. Thanks for the answer.
